docs/plugin-proposal-decorators.md

```diff
@@ -80,17 +80,31 @@ require("@babel/core").transformSync("code", {
 
 ## Options
 
-### `decoratorsBeforeExport`
-
-`boolean`
-
 <details>
   <summary>History</summary>
 | Version | Changes |
 | --- | --- |
-| `v7.2.0` | `decoratorsBeforeExport` must be specified. Before that it defaults to `false` |
+| `v7.17.0` | Added the `version` option with support for `"2021-12"`, `"2018-09"` and `"legacy"` |
 </details>
 
+### `version`
+
+`"2021-12"`, `"2018-09"` or `"legacy"`. Defaults to `"2018-09"`.
+
+Selects the decorators proposal to use:
+- `legacy` is the original Stage 1 one
```

Comment:
For reference here is the stage 1 spec: https://github.com/wycats/javascript-decorators/blob/e1bf8d41bfa2591d949dd3bbf013514c8904b913/README.md